dashboard/test/models/experiment_test.rb

```diff
@@ -94,6 +94,17 @@ class ExperimentTest < ActiveSupport::TestCase
       section_id: section.id + 1
     assert_empty Experiment.get_all_enabled(section: section)
   end
+
+  test 'single user experiment is enabled' do
+    facilitator_yes = User.first || create(:facilitator)
+    facilitator_no = User.second || create(:facilitator)
+    experiment = SingleUserExperiment.first || create(:single_user_experiment, min_user_id: facilitator_yes.id)
```

Comment:
Also, why are we creating facilitators? Doing so suggests to me that somehow the test should depend on the user being a facilitator (which it seemingly does not).